clar.c

```diff
@@ -145,7 +151,7 @@ static struct {
 
 	int report_errors_only;
 	int exit_on_error;
-	int report_suite_names;
+    int verbosity;
```

Comment:
Indent is funny here.

Reply:
Thanks for catching this, my editor on my laptop has started doing stupid things and this one has been tricky to track down. 😠